document/web-api/index.bs

```diff
@@ -150,6 +152,8 @@ The [=serialization steps=], given |value|, |serialized|, and |forStorage|, are:
     1. Set |serialized|.\[[Bytes]] to the [=sub-serialization=] of |value|.\[[Bytes]].
 
     1. Set |serialized|.\[[AgentCluster]] to the [=current Realm=]'s corresponding [=agent cluster=].
+    
+    1. Set |serialized|.\[[Origin]] to be the [=origin-serialization=] of the user agent's [=origin=].
```

Comment:
I'm also wondering if using the serialization is correct - this would perhaps allow transferring between two opaque origins. I'm not sure if we can store the origin itself here - @annevk?